src/tests/end-to-end/tests/target-page/visualization-boxes.test.ts

```diff
@@ -19,21 +19,18 @@ describe('Target Page visualization boxes', () => {
         await popupPage.gotoAdhocPanel();
     });
 
-    beforeEach(async () => {
-        await popupPage.disableAllToggles();
-        await targetPage.waitForVisualizationBoxesToDisappear();
-    });
-
     afterAll(async () => {
         if (browser) {
             await browser.close();
             browser = undefined;
         }
     });
 
-    const adhocTools = ['Automated checks', 'Headings', 'Landmarks'];
+    const adhocTools = ['Automated checks', 'Headings'];
```

Comment:
Did you try reordering these and testing for the same class of flakiness? I'm suspicious about whether it's actually an issue with Landmarks as opposed to being "the third one to run" or "the one after Headings".

Reply:
I did not, but thats a good point.

Reply:
Also, I took out `Landmarks` because it showed to be the most failed test in past 15 days. I am still going to experiment with the changes that you mentioned to see whats the reason behind it.